eks-distro-base/Makefile

```diff
@@ -39,6 +39,8 @@ images:
 		--local dockerfile=./ \
 		--local context=. \
 		--output type=image,oci-mediatypes=true,name=$(IMAGE),push=true
```

Comment:
Does this `$(IMAGE)` use the date epoch tag from check_update?

Reply:
Yes, wherever we're calling make release for EKS Distro base, i.e. periodic and postsubmit, we're overriding the IMAGE_TAG with the date epoch tag.